app/src/test/java/org/simple/clinic/scanid/ScanSimpleIdEffectHandlerTest.kt

```diff
@@ -52,4 +55,32 @@ class ScanSimpleIdEffectHandlerTest {
     ))
     verifyZeroInteractions(uiActions)
   }
+
+  @Test
+  fun `when parse json into patient prefill info object effect is received, then parse the json`() {
+    // given
+    val expectedJson = """
+    {
+    "hidn":"28-3123-2283-6622",
+    "hid":"mogithduraisamy@ndhm",
+    "name":"Mogith",
+    "gender":"M",
+    "statelgd":"34",
+    "distlgd":"600",
+    "dob":"25/6/2012",
+    "address":"No.42,
+     eswaran kovil street"
+     }
+     """
+    val adapter = moshi.adapter(PatientPrefillInfo::class.java)
+    val deserialized: PatientPrefillInfo? = adapter.fromJson(expectedJson)
```

Comment:
Let's rename this to `patientPrefillInfo` or whatever the deserialized object name is.